tutel/impls/moe_layer.py

```diff
@@ -167,10 +167,11 @@ def __init__(
                 assert 'fused_custom_fn' not in experts, "`fused_custom_fn` option for Tutel Moe-layer has been deprecated, please follows helloworld_from_scratch.py for custom construction instead."
                 assert 'implicit_dropout_p' not in experts, "`implicit_dropout_p` option for Tutel Moe-layer has been deprecated, please use torch.nn.Dropout(p=implicit_dropout_p) on custom activation_fn (for fc1_dropout) and after Tutel Moe-layer (for fc2_dropout) instead."
 
+            experts['model_dim'] = self.model_dim
+            experts['local_experts'] = self.num_local_experts
+            experts['sharded_count'] = self.sharded_count
```

Comment:
How to explicitly set `bias=False`?

Reply:
`experts` arg passed into the `MOELayer.__init__` is treated as a  dict of kwargs which will be `**` expanded into the expert init below: `self.experts = fused_experts.ExpertModule(**experts)`.

This kwarg dict should set bias (if you want to set bias).

```
            experts['model_dim'] = self.model_dim
            experts['local_experts'] = self.num_local_experts
            experts['sharded_count'] = self.sharded_count
```
are set from within the context layer because they are defined in the MOELayer context therefore this is the correct place to set these.

Reply:
`has_fc1_bias` and `has_fc2_bias` both default to True.